Resources/Prototypes/DeltaV/Entities/Clothing/Uniforms/jumpsuits.yml

```
@@ -140,3 +140,47 @@
     sprite: DeltaV/Clothing/Uniforms/Jumpsuit/sober.rsi
   - type: Clothing
     sprite: DeltaV/Clothing/Uniforms/Jumpsuit/sober.rsi
+
+- type: entity
+  parent: ClothingUniformBase
+  id: ClothingUniformJumpsuitLostTourist
+  name: lost tourist uniform
+  description: The uniform of those who just want to relax on a beach... Where is the beach anyway?
+  components:
+  - type: Sprite
+    sprite: DeltaV/Clothing/Uniforms/Jumpsuit/lost_tourist.rsi
+  - type: Clothing
+    sprite: DeltaV/Clothing/Uniforms/Jumpsuit/lost_tourist.rsi
+
+- type: entity
+  parent: ClothingUniformBase
+  id: ClothingUniformJumpsuitJeansRed
+  name: red sweater with jeans
+  description: A red sweater with blue jeans. This one has the dominant jeans.
+  components:
+  - type: Sprite
+    sprite: DeltaV/Clothing/Uniforms/Jumpsuit/jeans_red.rsi
+  - type: Clothing
+    sprite: DeltaV/Clothing/Uniforms/Jumpsuit/jeans_red.rsi
+
+- type: entity
+  parent: ClothingUniformBase
+  id: ClothingUniformJumpsuitJeansGreen
+  name: green sweater with jeans
+  description: A green sweater with blue jeans. This one has the recessive jeans.
+  components:
+  - type: Sprite
+    sprite: DeltaV/Clothing/Uniforms/Jumpsuit/jeans_green.rsi
+  - type: Clothing
+    sprite: DeltaV/Clothing/Uniforms/Jumpsuit/jeans_green.rsi
+    
+- type: entity
+  parent: ClothingUniformBase
+  id: ClothingUniformJumpsuitJeansBrown
+  name: brown sweater with jeans
+  description: A brown sweater with blue jeans.
+  components:
+  - type: Sprite
+    sprite: DeltaV/Clothing/Uniforms/Jumpsuit/jeans_brown.rsi
+  - type: Clothing
+    sprite: DeltaV/Clothing/Uniforms/Jumpsuit/jeans_brown.rsi
```

Missing newline at the end of the file
```suggestion
    sprite: DeltaV/Clothing/Uniforms/Jumpsuit/jeans_brown.rsi

```